### DISCUSSION TITLE
兴化语记（后端）讨论区

### DISCUSSION THREAD

**Discussion by lin594** [e-dialect/hinghwa-dict-backend#255] [Announcements]
## 👋 你好！
  我们在 GitHub 讨论区中广泛吸收建议与意见，欢迎大家留言：
  * 提出你所关注的问题
  * 分享想法
  * 与其他社区成员一起讨论

群策群力铸就成功 💪

**Comment by wgbao884120** [comment]
您好！我们是莆仙话（兴化话）方言技术项目组，正在做莆仙方言的语音数据采集与人工智能模型研发（语音识别 ASR、语音合成 TTS）。我们非常钦佩兴化语记在莆仙方言数字化上所做的工作——贵站收录的词语、标准 IPA 注音、拼音方案以及配套发音音频，是目前最系统、最权威的莆仙话公开资源之一。

我们希望能与贵团队正式沟通数据使用授权事宜，或者合作事宜，而非未经许可直接使用。具体请求如下：

请求事项

1. **全量数据导出**：希望获得兴化语记词典数据（words 词条表、characters 单字读音表，含标准 IPA、标准拼音、释义、普通话对照）的完整导出文件；
2. 发音音频包**：希望获得已收录词条配套发音音频（mp3/m4a，当前约 958 条）的下载包；
3. **使用授权**：在上述数据基础上，授权我方用于——
   - 莆仙方言语音识别（ASR）与语音合成（TTS）模型的**训练与评测**；
   - 训练所得模型的**商业化应用**（包括但不限于方言翻译、语音助手、学习应用）。
